tools/Tool.cc

```diff
@@ -51,6 +58,53 @@ static bool outputIsColor() {
     return sColor;
 }
 
+static bool tokenize(const char* input, deque<string> &args)
+{
+    if(input == nullptr) {
+        return false;
+    }
+    
+    // " (Matches a " character (ASCII 34)
```

Comment:
This will still match a double-quote preceded by a backslash, won't it? Meaning you can't escape quotes.

Reply:
You *can* escape quotes, just not unless they are also inside quotes (which in my mindset at the time was the only place that they would occur, oops).  This could probably be solved by adding a "not backslash" to each of the outer quotes